bin/varnishd/cache/cache.h

```diff
@@ -403,6 +403,7 @@ struct busyobj {
 	struct http		*bereq0;
 	struct http		*bereq;
 	struct http		*beresp;
+	struct objcore		*bereq_body;
```

Comment:
Same ABI problem since the field is not appended at the end of the struct.

Both could have been avoided, but probably fine with our stability promise.